plugins/poll/lib/polls_validator.rb

```diff
@@ -96,5 +124,32 @@ def valid_multiple_choice_settings?(poll)
 
       true
     end
+
+    def valid_numbers?(poll)
+      return true if poll["type"] != "number"
+
+      valid = true
+
+      min = (poll["min"].presence || 1).to_i
+      max = (poll["max"].presence || 2_147_483_647).to_i
+      step = (poll["step"].presence || 1).to_i
+
+      if min <= 0
+        @post.errors.add(:base, "Min " + I18n.t("errors.messages.greater_than", count: 0))
+        valid = false
+      end
+
+      if max <= 0
+        @post.errors.add(:base, "Max " + I18n.t("errors.messages.greater_than", count: 0))
+        valid = false
+      end
+
+      if step <= 0
```

Comment:
We should also validate that the `step` value allows for at least 1 number to be generated.